crest/Assets/Crest/Crest/Scripts/LodData/LodTransform.cs

```diff
@@ -129,7 +129,8 @@ public float MaxWavelength(int lodIdx)
             float oceanBaseScale = OceanRenderer.Instance.Scale;
             float maxDiameter = 4f * oceanBaseScale * Mathf.Pow(2f, lodIdx);
             float maxTexelSize = maxDiameter / OceanRenderer.Instance.LodDataResolution;
-            return 2f * maxTexelSize * OceanRenderer.Instance.MinTexelsPerWave;
+            float texelsPerWave = 2f;
```

Comment:
ShapeGerstnerBatched looks strange with 2.

![Screenshot (311)](https://user-images.githubusercontent.com/5249806/133220002-d284d9c9-8e45-48d5-b166-23ebaa2a25b5.png)
With 2

![Screenshot (310)](https://user-images.githubusercontent.com/5249806/133220311-bee5ba00-bcaa-45ce-8afe-97add2514c94.png)
With 3

Reply:
Yeah it's always been greater than 2 because it looked better for SGB, and this change degrades the quality. I propose we deprecate. SGB as a next step.